plugins/log_request_plugin/src/main.rs

```diff
@@ -71,13 +71,3 @@ async fn main() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
-
-/// does a one second pause.
-///
-/// NOTE: this is a quick-fix to enable step 4. in main to be executed.
-/// if `do_one_second_pause()` is not called, step 4. in main
-/// does not get triggered.
-// see #846
-async fn do_one_second_pause() {
```

Comment:
I am a bit confused with the name of the PR and the actual changes here?

Reply:
Just a mixed up commits